srcs/CGI/CGI.cpp

```diff
@@ -119,3 +129,20 @@ void	CGI::CloseAssign_(int *fd) {
 		*fd = -1;
 	}
 }
+
+std::string	CGI::MakeClientAddress_() {
```

Comment:
Just a small stupid comment, for me, it would be better to name this like:
```suggestion
std::string	CGI::BuildClientAddress_() {
```

Reply:
Very interesting, what are `REMOTE_ADDR` and `REMOTE_HOST` for? It's curious they point to the same address, why would you need this "duplicated" information?

Reply:
From [CGI RFC](https://datatracker.ietf.org/doc/html/rfc3875#section-4.1.9):

```
 4.1.9.  REMOTE_HOST

   The REMOTE_HOST variable contains the fully qualified domain name of
   the client sending the request to the server, if available, otherwise
   NULL.  Fully qualified domain names take the form as described in
   section 3.5 of RFC 1034 [17] and section 2.1 of RFC 1123 [12].
   Domain names are not case sensitive.

      REMOTE_HOST   = "" | hostname | hostnumber
      hostname      = *( domainlabel "." ) toplabel [ "." ]
      domainlabel   = alphanum [ *alphahypdigit alphanum ]
      toplabel      = alpha [ *alphahypdigit alphanum ]
      alphahypdigit = alphanum | "-"

   The server SHOULD set this variable.  If the hostname is not
   available for performance reasons or otherwise, the server MAY
   substitute the REMOTE_ADDR value.
```